File: FungeeCommandsManager/src/main/java/org/openredstone/commands/Derps.java
package org.openredstone.commands;

import net.md_5.bungee.api.CommandSender;
import net.md_5.bungee.api.chat.TextComponent;
import org.openredstone.handlers.DerpHandler;

import java.util.stream.IntStream;

public class Derps extends FunCommand {
    private final DerpHandler handler;

    public Derps(DerpHandler handler) {
        super("derps");
        this.handler = handler;
    }

    @Override
    public void execute(CommandSender sender, String[] args) {
        IntStream.range(0, handler.derpCount())
                .mapToObj(this::createDerpLine)
                .forEach(sender::sendMessage);
    }

    private TextComponent createDerpLine(int derpIndex) {
        return new TextComponent(
                derpIndex + ". " + handler.derpByIndex(derpIndex)
        );
    }
}

File: FungeeCommandsManager/src/main/java/org/openredstone/commands/Derps.kt
package org.openredstone.commands

import net.md_5.bungee.api.CommandSender
import net.md_5.bungee.api.chat.TextComponent
import org.openredstone.FungeeCommandsManager
import org.openredstone.handlers.DerpHandler

class Derps(plugin: FungeeCommandsManager, private val handler: DerpHandler) : FunCommand(plugin, "derps") {
    override fun execute(
        sender: CommandSender,
        args: Array<String>
    ) {
        (0 until handler.derpCount())
            .map(this::createDerpLine)
            .forEach(sender::sendMessage)
    }

    private fun createDerpLine(derpIndex: Int): TextComponent =
        TextComponent(
            derpIndex.toString() + ". " + handler.derpByIndex(derpIndex)
        )

}
